.gitlab-ci.yml

```diff
@@ -263,6 +261,7 @@ LiveFallback:
 # Publish static assets
 Pub::LiveFallback:
   extends: .publish_assets
+  stage: deploy-live
```

Comment:
Both `Pub::Live` and `Pub::LiveFallback` extend the `.publish_assets` template, which has `stage: deploy-live` defined in it.

If you want to move the stage from the template to the jobs, as you did for Live and LiveFallback, then you should probably delete the stage from the `.publish_assets` template and add it to `Pub::Live`. Or leave `deploy-live` stage in `.publish_assets` and remove it from here 🤷